models/dex/dex_schema.yml

```diff
@@ -5,36 +5,39 @@ models:
     meta:
       blockchain: ethereum
       sector: dex
-      contributors: jeff-dude
+      contributors: jeff-dude, bh2smith
     config:
-      tags: ['ethereum','uniswap','trades', 'dex','cross-chain']
+      tags: ['ethereum', 'trades', 'dexes', 'aggregators', 'cross-chain']
```

Comment:
I found the fact that "uniswap" was tagged specifically in the "dex_trades" table to be biased and removed it. Unless we expect every protocol to tag themselves here as well.

Reply:
If we want to follow what was done with nft trades, we would want protocol to tag themselves in there but it might not be very easy to maintain so let's keep this as is